geode-docs/tools_modules/gfsh/command-pages/start.html.md.erb

```diff
@@ -692,7 +692,7 @@ See <a href="../../../configuring/cluster_config/gfsh_persist.html">Overview of
 </tr>
 <tr class="odd">
 <td><span class="keyword parmname">\-\-hostname-for-clients</span></td>
-<td>Sets the ip address or host name that this cache server is to listen on for client connections. Setting a specific hostname-for-clients causes server locators to use this value when telling clients how to connect to this cache server. This is useful in the case where the cache server may refer to itself with one hostname, but the clients need to use a different hostname to find the cache server. The value &quot;&quot; causes the bind-address to be given to clients. A null value will be treated the same as the default &quot;&quot;.</td>
+<td>Sets the IP address or host name that a locator will provide to clients. Clients use the address to connect to a server. Set this value when clients use a different address to connect with the server than the <code>bind-address</code>, as those clients might with servers in a private cloud or multi-homed environment. Not specifying this option or setting this option to the empty string (&quot;&quot;) causes the <code>bind-address</code> to be given to clients.</td>
```

Comment:
I think server-bind-address, if set, will be sent to clients instead of bind-address.